### PR TITLE
[11.0] [ADD] account_budget_template

### PR DESCRIPTION
**Pull request by dalonsod**
A budget template contains different budgetary positions and defines a periodicity
to create with a button the corresponding lines.

### CODE REVIEW

File: account_budget_template/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Account Budget Template",
+    "version": "11.0.1.2.0",
+    "category": "Accounting",
+    "license": "AGPL-3",
+    "author": "AvanzOSC,"
+              "Odoo Community Association (OCA)",
+    "website": "http://www.avanzosc.es",
```
Comment (nikul-serpentcs) [inline account_budget_template/__manifest__.py]: URL should be 'https://github.com/OCA/account-financial-tools'

**Review by nikul-serpentcs — COMMENTED** [review]
nitpicking

File: account_budget_template/__manifest__.py (hunk 1 above)
Comment (dalonsod) [reply account_budget_template/__manifest__.py]: Done!

**Comment by dalonsod** [comment]
Well, we realized there's a better repo target (account-budgeting). Close this PR

**Comment by oihane** [comment]
New PR in [account-budgeting#21](https://github.com/OCA/account-budgeting/pull/21/)

**Pull request closed without merging**